hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalogNew.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.hive;
+
+import java.util.Collections;
+import org.apache.iceberg.catalog.CatalogTests;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * Run all the tests from abstract of {@link CatalogTests} It should replace the other tests classes
+ * like {@link TestHiveCatalog}
+ */
+public class TestHiveCatalogNew extends CatalogTests<HiveCatalog> {
```

Comment:
I think we can keep this class name as as `TestHiveCatalog` as rename old file to `TestWithHiveCatalog` so that you can remove it in the follow up? Extra tests that are not present in the this file can be added to this file. 

Also it would be nice if we do this in this PR.